docs/setup.rst

```diff
@@ -0,0 +1,79 @@
+Setting Up Corobo
+=================
+
+The following documentation illustrates the process to install corobo and interface it with the gitter(or any other backend) API.
+
+Installation
+------------
+
+*  To install corobo, first you need to setup a virtual environment(it is not mandatory but highly recommended). This can be done as follows:
+   ::
+     pip3 install virtualenv
+     virtualenv venv -p python
+     cd venv
+     source bin/activate ---->  For Linux
+     \Scripts\activate   ---->  For Windows
+
+*  Next, you can clone the corobo package via git and install the requirements(dependent packages).
+   ::
+     git clone https://github.com/coala/corobo.git
+     pip install -r requirements.txt
+     
+     
+Configuring the backend
+------------------------
+
+#. Clone the err-backend-gitter repository in your "corobo" folder.
+   ::
+     git clone https://github.com/errbotio/errbot@a0f35732484c8c0692e123c48653517cffa21a42
+          
+
+#. To make the bot interact with the gitter API you will need a github account(this account can be used to login to gitter) to generate the gitter access token.
```

Comment:
You actually don't need a github account to login to gitter. One can use gitlab, twitter, or whatever else there is available.

Reply:
Also take care of the line length.